### PR TITLE
Add profile support for iOS

### PR DESCRIPTION
**Pull request by HugoMaster**

### CODE REVIEW

File: RNXMPP/RNXMPPService.m [hunk 1]
```diff
@@ -117,10 +118,10 @@ - (void)setupStream
     // The vCard Avatar module works in conjuction with the standard vCard Temp module to download user avatars.
     // The XMPPRoster will automatically integrate with XMPPvCardAvatarModule to cache roster photos in the roster.
 
-//    xmppvCardStorage = [XMPPvCardCoreDataStorage sharedInstance];
```
Comment (HugoMaster) [inline RNXMPP/RNXMPPService.m]: Don't know why this lines are seen in git as changed, i revert all changes made here.

**Comment by nicolai86** [comment]
test

**Comment by nicolai86** [comment]
I'm missing adjustments for the android side. Did you have a look at those, @HugoMaster ?

**Comment by HugoMaster** [comment]
Its in second PR

**Comment by HugoMaster** [comment]
Closing this, PR as #2 include all changes and its not need for two separate prs.

**Pull request closed without merging**